pkgs/applications/accessibility/mousetweaks/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, fetchFromGitLab
+, autoreconfHook, pkgconfig, makeWrapper
+, glib, gtk3, gsettings-desktop-schemas
+, libX11, libXtst, libXfixes, libXcursor
+}:
+stdenv.mkDerivation rec {
+  pname = "mousetweaks";
+  version = "3.32.0";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "GNOME";
+    repo = pname;
+    rev = version;
+    sha256 = "0acaqpskz15xzrxw0fqh72wps1i78ybxls1gfm1nr2frc246ibww";
+  };
+
+  nativeBuildInputs = [ autoreconfHook pkgconfig makeWrapper ];
+
+  buildInputs = [
+    glib gtk3 gsettings-desktop-schemas
+    libX11 libXtst libXfixes libXcursor
+  ];
+
+  postInstall = ''
+    wrapProgram "$out/bin/mousetweaks" \
+      --prefix XDG_DATA_DIRS : "$GSETTINGS_SCHEMAS_PATH:$out/share/gsettings-schemas/${pname}-${version}"
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Provides mouse accessibility enhancements for the GNOME desktop";
+    longDescription = ''
+      Mousetweaks provides mouse accessibility enhancements for the GNOME
+      desktop. These enhancements are:
+
+      - It offers a way to perform the various clicks without using any
+      physical mouse buttons. (Hover Click)
+
+      - It allows users to perform a secondary click by keeping the primary
+      mouse button pressed for a predetermined amount of time. (Simulated
+      Secondary Click)
+
+      The features can be activated and configured through the Universal Access
+      panel of the GNOME Control Center.
+    '';
+    homepage = https://wiki.gnome.org/Projects/Mousetweaks;
+    license = licenses.gpl2;
+    platforms = platforms.all;
+    maintainers = [ maintainers.johnazoidberg ];
+    repositories.git = https://gitlab.gnome.org/GNOME/mousetweaks;
```

Comment:
I do not think this does anything.

Reply:
I saw it on some other derivations and thought it couldn't hurt.
I'm not sure it does anything either :man_shrugging:

Reply:
72 derivations have it

Reply:
I suspect it comes from update walker, an old update system nobody uses.

Reply:
No reference to this in update walker. According to 452b25b6c33c04a745008457fcb9f8f45de054c0 it is used by some @Phreedom’s tool, possibly https://github.com/Phreedom/nixpkgs-monitor, which refers to it in its source code.

Reply:
Hmm, yeah I understand Eelco's concern but since there are already a lot of packages with it I think we can continue for packages that the repository is neither the homepage nor the source.

Reply:
Oh, I did not even notice the comments.

Reply:
Most of GNOME's wiki pages have a mention of the git repository, and even in this case where it's an old
url `https://git.gnome.org/browse/mousetweaks` it still redirects to GitLab.

There's not a lot of references to this attribute and I don't even see it documented anywhere. I'd say it's safe to not
use it, though I do see a place for it being useful as you mentioned @JohnAZoidberg.

Reply:
Well it is kind of formal, it's been a part of the valid `meta` attributes for quite a while.
@copumpkin added it here: https://github.com/NixOS/nixpkgs/commit/1a4ca220e15480818b78041d46bf0ca438671d33#diff-3c8a88619c3ca3d61eb0ce8794d9bb74R228
He added the check probably after realizing that lots of people started to use it.